pkg/config/sync.go

```diff
@@ -276,6 +290,14 @@ func logCurrentConfig() {
 			config = append(config, fmt.Sprintf("workspace.ignoredUnrecoverableEvents=%s",
 				strings.Join(Workspace.IgnoredUnrecoverableEvents, ";")))
 		}
+		if Workspace.DefaultStorageSize != nil {
+			if Workspace.DefaultStorageSize.Common != nil {
+				config = append(config, fmt.Sprintf("workspace.defaultStorageSize.%s=%s", constants.CommonStorageClassType, Workspace.DefaultStorageSize.Common.String()))
+			}
+			if Workspace.DefaultStorageSize.PerWorkspace != nil {
+				config = append(config, fmt.Sprintf("workspace.defaultStorageSize.%s=%s", constants.PerWorkspaceStorageClassType, Workspace.DefaultStorageSize.PerWorkspace.String()))
```

Comment:
We should hard-code the config names, this will result in logging e.g.
```
workspace.defaultStorageSize.per-workspace=5Gi
```
even though the actual config field is
```
workspace.defaultStorageSize.perWorkspace
```

```suggestion
				config = append(config, fmt.Sprintf("workspace.defaultStorageSize.common=%s", Workspace.DefaultStorageSize.Common.String()))
			}
			if Workspace.DefaultStorageSize.PerWorkspace != nil {
				config = append(config, fmt.Sprintf("workspace.defaultStorageSize.perWorkspace=%s", Workspace.DefaultStorageSize.PerWorkspace.String()))
```

Reply:
Sounds good, I went with `per-workspace` instead of `perWorkspace`. Let me know if you rather it be `perWorkspace`

Reply:
It should match the actual field in the config though -- the dot delimited format is meaningful here. If you try to set `workspace.defaultStorageSize.per-workspace` in the DevWorkspaceOperatorConfig, it will be invalid.